docs/fsharp/using-fsharp-on-azure/queue-storage.md

```diff
@@ -30,67 +30,59 @@ Next, use a [package manager](package-management.md) such as [Paket](https://fsp
+[!code-fsharp[QueueStorage](../../../samples/snippets/fsharp/azure/queue-storage.fsx#L1-L2)]
 
 ### Get your connection string
 
 You'll need an Azure Storage connection string for this tutorial. For more information about connection strings, see [Configure Storage Connection Strings](/azure/storage/storage-configure-connection-string).
 
 For the tutorial, you'll enter your connection string in your script, like this:
 
-[!code-fsharp[QueueStorage](~/samples/snippets/fsharp/azure/queue-storage.fsx#L9-L9)]
+[!code-fsharp[QueueStorage](../../../samples/snippets/fsharp/azure/queue-storage.fsx#L8-L8)]
 
 However, this is **not recommended** for real projects. Your storage account key is similar to the root password for your storage account. Always be careful to protect your storage account key. Avoid distributing it to other users, hard-coding it, or saving it in a plain-text file that is accessible to others. You can regenerate your key using the Azure portal if you believe it may have been compromised.
 
 For real applications, the best way to maintain your storage connection string is in a configuration file. To fetch the connection string from a configuration file, you can do this:
 
-[!code-fsharp[QueueStorage](~/samples/snippets/fsharp/azure/queue-storage.fsx#L11-L13)]
+[!code-fsharp[QueueStorage](../../../samples/snippets/fsharp/azure/queue-storage.fsx#L10-L12)]
 
 Using Azure Configuration Manager is optional. You can also use an API such as the .NET Framework's `ConfigurationManager` type.
 
-### Parse the connection string
-
-To parse the connection string, use:
-
-[!code-fsharp[QueueStorage](~/samples/snippets/fsharp/azure/queue-storage.fsx#L19-L20)]
-
-This will return a `CloudStorageAccount`.
-
 ### Create the Queue service client
 
-The `CloudQueueClient` class enables you to retrieve queues stored in Queue storage. Here's one way to create the service client:
+The `QueueClient` class enables you to retrieve queues stored in Queue storage. Here's one way to create the service client:
 
-[!code-fsharp[QueueStorage](~/samples/snippets/fsharp/azure/queue-storage.fsx#L26-L26)]
+[!code-fsharp[QueueStorage](../../../samples/snippets/fsharp/azure/queue-storage.fsx#L19-L19)]
 
 Now you are ready to write code that reads data from and writes data to Queue storage.
 
 ## Create a queue
 
 This example shows how to create a queue if it doesn't already exist:
 
-[!code-fsharp[QueueStorage](~/samples/snippets/fsharp/azure/queue-storage.fsx#L32-L36)]
+[!code-fsharp[QueueStorage](../../../samples/snippets/fsharp/azure/queue-storage.fsx#L25-L25)]
 
 ## Insert a message into a queue
 
 To insert a message into an existing queue, first create a new
-`CloudQueueMessage`. Next, call the `AddMessage` method. A
-`CloudQueueMessage` can be created from either a string (in UTF-8
-format) or a `byte` array, like this:
+Message. Next, call the `SendMessage` method. A
+Message can be created from string (in UTF-8
+format), like this:
```

Comment:
It might be worth showing off different examples with [BinaryData](https://docs.microsoft.com/en-us/dotnet/api/overview/azure/system.memory.data-readme) because JSON is the most common customer use case for messages.